.github/workflows/proto.yml

```
@@ -40,3 +44,17 @@ jobs:
         run: make proto-check-breaking PR_TARGET_REPO=https://github.com/${{ github.repository }}.git PR_TARGET_BRANCH=${{ github.base_ref }}
         working-directory: golang/cosmos
         if: env.GIT_DIFF
+  up-to-date:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - uses: ./.github/actions/restore-golang
+        with:
+          go-version: '1.20'
+      - name: check protobuf definitions are up-to-date
+        run: |
+          cd packages/cosmic-proto
+          ./update-protos.sh
+          test -z "$(git status --porcelain)"
```

Might be nice to print out some diagnostics:

```suggestion
          out=$(git status --porcelain)
          test -n "$out" || exit 0
          echo "Unexpected git status output:"
          echo "$out"
          echo "Please run (cd packages/cosmic-proto && ./update-protos.sh)"
          echo "and commit the result."
          exit 1
```